docs/interpreting-experiments/confidence-intervals.md

```diff
@@ -1,34 +1,266 @@
+  } 
+  + 
+  \frac{
+    \mu_{data}
+  }{
+    \sigma^2_{data}
+  }
+}{
+  \frac{1}{\sigma^2_{prior}}
+  +
+  \frac{1}{\sigma^2_{data}}
+}
+
+\\[20pt]
+
+
+=&~~ \frac{1}{\sigma^2_{prior} + \sigma^2_{data}} \left(
+  \sigma^2_{prior}\mu_{data} + \sigma^2_{data}\mu_{prior}
+\right)
+
+&\text{\it\small{{(multiply top and bottom by $\sigma^2_{prior}\sigma^2_{data}$)}}}
+
+\\[10pt]
+
+\sigma^2_{post} =&~~ \frac{1}{\frac{1}{\sigma^2_{prior}} + \frac{1}{\sigma^2_{data}}}
+\end{align}
+$$
+
+Following [Gelman](http://www.stat.columbia.edu/~gelman/book/BDA3.pdf) (p50) (with a slight tweak), we can rewrite the posterior mean $\mu_{post}$ as:
+
+$$
+\begin{equation}
+\mu_{post} = \mu_{data} - \frac{\sigma^2_{data}}{\sigma^2_{data} + \sigma^2_{prior}}
+(\mu_{data} - \mu_{prior})
+\end{equation}
+$$
+
+If we note that:
+
+- $\mu_{data}$ is the lift observed in the data (i.e. $E[\hat{\mathcal{L}}] = \frac{m_C}{m_T} - 1$)
+- $\frac{\sigma^2_{data}}{\sigma^2_{data} + \sigma^2_{prior}}$ represents how spread out the data is compared to the prior; if the variance of our lift is very low (because for example we have a lot of data), then this quantity will tend toward $0$[^8]
+- $(\mu_{data} - \mu_{prior})$ is the difference between what we've observed and what we expected to observe before we started the experiment
+
+Then we can restate equation (8) as: the lift we observed in the experiment, *shrunk* toward the lift we *expected* to see (according to the prior) by an amount that is set by how precise our lift estimate is relative to the strength of our prior.
+
+[^8]: And it will get to zero faster if our prior is *weaker*, that is, if it has a higher variance. If we have a *strong* belief, represented by a *low* value of $\sigma^2_{prior}$, then moving this term toward zero requires need *more* data (or, data that doesn't vary much).
+
+#### Confidence Intervals
+
+For fixed-sample and Bayesian analysis, constructing a normal distribution that describes the average lift (equation 1-3 in the case of fixed-sample, and the posterior from equations 5-6 in the case of Bayesian) is enough to construct confidence intervals: the lower and upper bounds for significance level $\alpha$ (e.g. 5%) are simply the $\frac{\alpha}{2}^{th}$ and $(1 - \frac{\alpha}{2})^{th}$ percentiles of that distribution.
+
+In order to guarantee always-valid confidence intervals for sequential analysis, however, we need to do something fancier. Specifically, we use the method described in [Howard et al. (2021)](https://arxiv.org/abs/1810.08240) to construct a bound such that the probability that the mean will be outside that bounds at any point is less than or equal to a significance level $\alpha$. In particular, we define the *confidence sequence* (that is, a sequence of confidence intervals) around the mean using equation (14) from the reference (with some changes in notation):
+
+$\hat \mu \pm \frac{\sigma}{t} \sqrt{ (t + \rho) \log \left( \frac{t+\rho}{\rho\alpha^2} \right)}$
```

Comment:
@schmit Do you remember where the sigma/t comes from, before the sqrt? I've skimmed the paper again but I'm assuming I'm just missing a notation thing

Reply:
IIRC in the paper they analyze a sum of $N(\mu,1)$ distributed random variables, but for us it's $N(\mu, \sigma^2)$ and then we are looking at the variance of the mean, not the sum so that's why we divide by $t$.